resources/linux-config-x86_64

```diff
@@ -1227,7 +1227,7 @@ CONFIG_VIRTIO_BLK=y
 #
 CONFIG_NVME_CORE=y
 CONFIG_BLK_DEV_NVME=y
-# CONFIG_NVME_MULTIPATH is not set
+CONFIG_NVME_MULTIPATH=y
```

Comment:
Great, but could you elaborate a little bit in the commit message why we need this?

Reply:
Of course. Commit message updated.